src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+import duke.command.*;
+
+public class Parser {
+    private String command;
+    
+    public Parser(String command) {
+        this.command = command;
+    }
+    
+    public Command parse() {
```

Comment:
Perhaps try not to write deeply nested loops in your function? You might consider using 'switch' for this function?☺️